torcharrow/functional.py

```diff
@@ -141,6 +141,60 @@ def add_tokens(
     return _dispatch("add_tokens", input_col, tokens, begin)
 
 
+def bpe_tokenize(
+    tokenizer,
+    input_col: Union[ListColumn, List[str]],
+) -> ListColumn:
+    """
+    Tokenize text into a list of tokens using the GPT-2 BPE Tokenizer
+
+    Parameters
+    ----------
+    tokenizer: An instance of the `GPT2BPEEncoder` class
+    input_col: List of input text
+
+    Examples
+    --------
+    >>> import torcharrow as ta
+    >>> from torcharrow import functional
+    >>> # init_bpe_encoder() is helper function that creates an instance of the `GPT2BPEEncoder` class
```

Comment:
should we show instantiation of _ta.GPT2BPEEncoder like you did for _ta.Vocab? Or alternatively give reference to init_bpe_encoder function from here https://github.com/pytorch/torcharrow/blob/cecbd9222dca54784b21c6e914869c90b0fec60e/torcharrow/test/transformation/test_text_ops.py#L59?